commands/copy.go

```diff
@@ -53,12 +54,24 @@ func (r *copyRunner) run(_ *cobra.Command, _ []string) error {
 	client := &http.Client{
 		Timeout: r.timeout,
 	}
+
+	var password string
+
 	if r.password != "" {
+		password = r.password
+	} else if os.Getenv(pbgopyPasswordFileEnv) != "" {
+		password, err = getPasswordFromEnv(os.Getenv(pbgopyPasswordFileEnv))
+		if err != nil {
+			return err
+		}
+	}
+
+	if len(password) != 0 {
```

Comment:
Why not just compare with "".
```suggestion
	if password != "" {
```